<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDevCfgWidget</class>
 <widget class="QWidget" name="CDevCfgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CDevCfgWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="pCfgTreeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>配置选项</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>设备信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>算法参数</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>级联配置</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视频参数</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="pNameLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>设备名:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="pConfigWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="pDevInfoPage">
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>259</width>
         <height>333</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>设备UUID：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="pDevUUIDEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>设备名称：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="pDevNameEdit">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>设备类型：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="pDevTypeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>软件版本：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="pSoftVerEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>硬件版本：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="pHardVerEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>算法版本：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="pAlgVerEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QCheckBox" name="pDHCPCBox">
          <property name="text">
           <string>DHCP开关</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>IP地址：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QLineEdit" name="pIpAddrEdit"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>掩码：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QLineEdit" name="pNetmaskEdit"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>网关：</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QLineEdit" name="pGatewayEdit"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>DNS：</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="2">
         <widget class="QLineEdit" name="pDnsEdit"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>HTTP端口：</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1" colspan="2">
         <widget class="QLineEdit" name="pHttpPortEdit"/>
        </item>
        <item row="12" column="2">
         <widget class="QPushButton" name="pSetCfgButton">
          <property name="text">
           <string>配置</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QPushButton" name="pGetCfgButton">
          <property name="text">
           <string>获取</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
